src/MLJCatBoostInterface.jl

```diff
@@ -28,6 +28,40 @@ function mlj_to_kwargs(model)
                             for name in fieldnames(typeof(model)))
 end
 
+"""
+    get_dtype_feature_ix(X, dtype)
+
+Get the index of the columns with a specific dtype
+
+Parameters
+----------
+- `X`: Table
+- `dtype`: DataType 
+
+Returns
+-------
+- `Vector{Int64}`
+"""
+function get_dtype_feature_ix(X, dtype)
+    return findall(eltype.(MMI.scitype.(eachcol(X))) .<: dtype)
+end
+
+"""
+Get cat features for model
+get_cat_features
+"""
+function prepare_input(X)
+    order_factor_ix = get_dtype_feature_ix(X, OrderedFactor)
+    for col in names(X)[order_factor_ix]
```

Comment:
What is this method `names` ? I think the method of that name in Tables.jl is only for named tuple tables, and matrix tables. You could do `MMI.schema(X).names`, which amounts to something like `Tables.schema(Tables.rowtable(X)).names` or `Tables.columnnames(Tables.rowtable(X))`. Unfortunately Tables.jl does not have a `names` method that can be directly applied to any table, as I understand it.